conans/client/rest/rest_client_v2.py

```diff
@@ -58,12 +58,14 @@ def get_recipe_manifest(self, ref):
         if not ref.revision:
             ref = self.get_latest_recipe_revision(ref)
         url = self.router.recipe_manifest(ref)
-        content = self._get_remote_file_contents(url)
+        cache = (ref.revision != "0")
```

Comment:
In the sources, when we refer to this _magic number_ we use `DEFAULT_REVISION_V1`

```suggestion
        cache = (ref.revision != DEFAULT_REVISION_V1)
```